server/src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -67,16 +66,28 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   /** Stores a new {@link Message}. */
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+
     UserService userService = UserServiceFactory.getUserService();
     if (!userService.isUserLoggedIn()) {
       response.sendRedirect("/");
       return;
     }
 
     String user = userService.getCurrentUser().getEmail();
-    String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
 
-    String userText = Jsoup.clean(request.getParameter("text"), Whitelist.none());
+
+    String userText = Jsoup.clean(request.getParameter("text"), Whitelist.basicWithImages());
```

Comment:
You don't need to clean here. You can leave this as 

`String userText = request.getParameter("text");`